src/DeepNgToDo/Tests/Frontend/angular/services/DeepNgLoginService.spec.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+import moduleName from '../../../../Frontend/js/app/angular/name';
+
+
+describe('Services', function() {
+  let deepNgLoginService;
+  let $q;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).